VhostConfigurationFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ola\RabbitMqAdminToolkitBundle;
+
+use Http\Client\Common\Plugin\ErrorPlugin;
+use Http\Client\Common\PluginClient;
+use Http\Discovery\HttpClientDiscovery;
+use RabbitMq\ManagementApi\Client;
+
+class VhostConfigurationFactory
+{
+    private ClientFactory $clientFactory;
+
+    private bool $deleteAllowed;
+
+    private array $connections;
+
+    private array $vhosts;
+
+    public function __construct(ClientFactory $clientFactory, bool $deleteAllowed, array $connections, array $vhosts)
+    {
+        $this->clientFactory = $clientFactory;
+        $this->deleteAllowed = $deleteAllowed;
+        $this->connections = $connections;
+        $this->vhosts = $vhosts;
+    }
+
+    public function getVhostConfiguration(string $vhostName): VhostConfiguration
```

Comment:
We probably don't care, but this would instanciate a new `Client` and a new `VhostConfiguration` every time a `VhostConfiguration` is requested instead of relying on a service. I understand it is the point to avoid loading those services. But maybe would it be worth mentionning why we do so in a comment somewhere ?

Reply:
Yeah actually this bundle is meant to be used as a one-shot, so it's better instantiating all the needed clients when we need it instead of doing it at every request :)

Reply:
And there's one command, that will call `getClient` once per vhost so no big deal I guess